homeassistant/components/ohme/strings.json

```diff
@@ -0,0 +1,51 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Configure your Ohme account.",
+        "data": {
+          "email": "[%key:common::config_flow::data::email%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "data_description": {
+          "email": "Enter the email address associated with your Ohme account. This should not be a social account (eg. Google).",
```

Comment:
How do you mean should not be a social account?

Reply:
As in if you signed in with Google when you created your account, you need to reset your password to use the integration. There's a note about this in the docs

Reply:
Instead of explaining this in the data_description, I would just move the social part to the description of the step itself.

"This has to be an Ohme account. If you signed up with Ohme with a third party account like Google, please reset your password via Ohme, so you can create your own account"

I am probably cutting edges, but let's make it a little more descriptive. Bonus points if we can have a separate place on the integration docs where we explain this